src-pyloid/server.py

```
@@ -553,3 +556,187 @@ async def clear_model_cache():
+@server.method()
+async def recordings_preview_state():
+    return get_controller().meetings.preview_state()
+
+
+@server.method()
+async def recordings_list(*, limit: int = 100, offset: int = 0, search: Optional[str] = None):
+    return get_controller().meetings.list_recordings(limit=limit, offset=offset, search=search)
+
+
+@server.method()
+async def recordings_get(*, id: int):
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)